deploy/runtime/defaults/PVWatts_None.txt

```diff
@@ -322,7 +322,8 @@ phoenix_az_33.450495_-111.983688_psmv3_60_tmy.csv
 solar_data_file_name_load
 1
 4
-0
+49
+phoenix_az_33.450495_-111.983688_psmv3_60_tmy.csv
```

Comment:
@dguittet @cpaulgilman should solar_data_file_name and solar_data_file_name_load not have a default string? The changes are not showing on most files because the default weather file is already loaded into the defaults file.

Reply:
Ah, I guess that's fine if there's a default file_name in the UI, especially if that's already how it is. What I need to avoid is SSC file names variables being stored, so `solar_resource_file` must be empty.

Reply:
I'm assuming that applies to every config with solar_resource_file? It seems that even the CSP configs have the default string. I just pushed that change for PV configs

Reply:
Ah, well, it's unavoidable then :D I'm okay with leaving them for the CSP and others. Thanks for fixing those!

Reply:
I can push a commit blanking out solar_resource_file for all configs that have it if you'd like. 88 additional default files from the ones already in this PR but its all the same change.

Reply:
Thanks Matt but I think it's not worth the time, unless you have already an automated way to do it?

Reply:
If not then don't worry about it! I've blanked it out before a year or so ago and it didn't last. It doesn't really create a bug, it's just I prefer users, when they try to run a default case in PySAM, to get an error about "you need to add a solar resource file" rather than "C:\dguittet\SAM\weatherfile\x.csv" doesn't exist.

Reply:
I used the Defaults Manager on every single configuration. It should have modified the default in configs where solar_resource_file did exist and not do anything to those that do not have it. You are right though it could easily get overwritten again. 

![image](https://user-images.githubusercontent.com/54449384/127053440-b5fc1d28-4d5a-45ea-9b45-859c5031cb56.png)